components/Analytics.tsx

```diff
@@ -1,9 +1,27 @@
-import { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react'
 import Router from 'next/router'
-import ReactGA from 'react-ga4';
+import ReactGA from 'react-ga4'
 import { isMobile } from 'react-device-detect'
 
-const trackingId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS;
+const trackingId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS
+
+function initBraveAnalytics() {
```

Comment:
```suggestion
function putUtmLabelsToCookies() {
```

Reply:
I would keep the current name but will add a comment `Put UTM labels from query-string to cookies`